src/main/java/com/uber/cadence/samples/hello/HelloException.java

```diff
@@ -80,101 +87,99 @@
  *     at com.uber.cadence.internal.worker.POJOActivityImplementationFactory$POJOActivityImplementation.execute(POJOActivityImplementationFactory.java:162)
  *
  * </pre>
- * Note that there is only one level of wrapping when crossing logical process boundaries. And that wrapper exception
- * adds a lot of relevant information about failure.
- * <p>
- * {@link IOException} is a checked exception. The standard Java way of adding <code>throws IOException</code> to
- * activity, child and workflow interfaces is not going to help. It is because at all levels it is never received directly, but
- * in wrapped form. Propagating it without wrapping would not allow adding additional context information
- * like activity, child workflow and parent workflow  types and IDs. The Cadence library solution is to provide special
- * wrapper method {@link Workflow#wrap(Exception)} which wraps a checked exception in a special runtime exception.
- * It is special because framework strips it when chaining exception across logical process boundaries. In this example
- * IOException is directly attached to ActivityFailureException besides being wrapped when rethrown.
- * </p>
+ *
+ * Note that there is only one level of wrapping when crossing logical process boundaries. And that
+ * wrapper exception adds a lot of relevant information about failure.
+ *
+ * <p>{@link IOException} is a checked exception. The standard Java way of adding <code>
+ * throws IOException</code> to activity, child and workflow interfaces is not going to help. It is
+ * because at all levels it is never received directly, but in wrapped form. Propagating it without
+ * wrapping would not allow adding additional context information like activity, child workflow and
+ * parent workflow types and IDs. The Cadence library solution is to provide special wrapper method
+ * {@link Workflow#wrap(Exception)} which wraps a checked exception in a special runtime exception.
+ * It is special because framework strips it when chaining exception across logical process
```

Comment:
because framework -> because the framework
chaining exception -> chaining exceptions